scripts/createAndApproveZendeskTicket.ts

```diff
@@ -51,7 +51,7 @@ const generateTicketData = (): ZendeskRequestData => ({
 const createAndApproveTicket = async () => {
   checkEnvironmentVariablesSet([
     'FIXED_SUBJECT_LINE',
-    'ZENDESK_RECIPIENT_EMAIL',
+    'FIXED_RECIPIENT_EMAIL',
```

Comment:
Leave this as `ZENDESK_RECIPIENT_EMAIL` please, that's maybe moved on or something since you made this PR.